vl-select.src.js

```diff
@@ -86,6 +171,15 @@ export class VlSelect extends NativeVlElement(HTMLSelectElement) {
     return choices;
   }
 
+  /**
+   * Geef de 'js-vl-select' wrapper terug dat door de dress functie wordt gegenereerd
+   * wordt.
+   * @return {null|*|Element} geeft 'js-vl-select' div terug of 'null' als de component nog niet geinitialiseerd is door 'dress()'
+   */
+  get _jsVlSelect() {
```

Comment:
Ik heb de _jsVlSelect functie hernoemd naar _wrapperElement omdat het dan duidelijker is wat de functie geeft.